### ISSUE TITLE
bug: dragging window to monitor triggers refocus on previous window from that monitor

### ISSUE THREAD

**Issue by theol0403** [paperwm/PaperWM#558]
**Describe the bug**
Often, when I drag a window to a second monitor with my mouse, and select the rightmost location for the window to go, the window will move to the correct location, but then the window to the left will scroll into focus, hiding the new window.

**To Reproduce**
1. Full width app on monitor 2
2. Medium width app on monitor 1
3. Drag app from monitor 1 to 2, and drop it on the right edge of the screen
4. Move the mouse one pixel

**Expected behavior**
The new window is visible on monitor 2.
Instead, the full width app gets focused and centred on monitor 2. 

**System information:**
Please execute `./gather-system-info.sh` in you PaperWM clone and paste the output below.

```
Distribution: Fedora Linux
GNOME Shell 44.2
Display server: Wayland
PaperWM branch/tag: develop
PaperWM commit: 249f096e7aa674850bd793b890179125d2d0bf3b
Enabled extensions:
- paperwm@hedning:matrix.org
- cpufreq@konkor
- clipboard-indicator@tudmotu.com
- bluetooth-quick-connect@bjarosze.gmail.com
- espresso@coadmunkee.github.com
- dash-to-panel@jderose9.github.com
- unite@hardpixel.eu
- fullscreen-hot-corner@sorrow.about.alice.pm.me
- Vitals@CoreCoding.com
- notification-timeout@chlumskyvaclav.gmail.com
- appindicatorsupport@rgcjonas.gmail.com
```

**Additional context**
I think this started happening with https://github.com/paperwm/PaperWM/pull/544

**Comment by jtaala** [comment]
Thanks @theol0403 - will hopefully get some time to look at this given I'm (hopefully) wrapping up #556.

**Comment by jtaala** [comment]
Hey @theol0403 - have fixed this bug in #556.  We weren't managing drag-drop windows to different monitors very well at all.

You might have noticed that when you drag a window to a new monitor there's a delay, until you move your mouse a bit or click before the that monitor is activated.  This is because the normal signals (mouse-move, click) that activate another monitor weren't catching for drag-drop windows.

Anyways, we now handle this correctly (in #556).  Can you give this a test?

You'll need to checkout the [submission-to-extensions.gnome.org](https://github.com/paperwm/PaperWM/tree/submission-to-extensions.gnome.org) branch and re-run the `install.sh` script:

```
git checkout submission-to-extensions.gnome.org
./install.sh
```
and then logout / login.

Jay.

**Comment by theol0403** [comment]
Fixed, thanks!!
